client/tests/integration/triggers/by_call_trigger.rs

```diff
@@ -266,10 +268,83 @@ fn trigger_should_be_able_to_modify_its_own_repeats_count() -> Result<()> {
 }
 
 #[test]
-fn unregister_trigger() -> Result<()> {
+fn only_account_with_permission_can_register_trigger() -> Result<()> {
+    // Building a configuration
     let (_rt, _peer, test_client) = <PeerBuilder>::new().with_port(10_035).start_with_runtime();
     wait_for_genesis_committed(&vec![test_client.clone()], 0);
 
+    let alice_account_id = AccountId::from_str("alice@wonderland")?;
+    let rabbit_account_id = AccountId::from_str("rabbit@wonderland")?;
+    let rabbit_keys = KeyPair::random();
+    let rabbit_account = Account::new(rabbit_account_id.clone(), rabbit_keys.public_key().clone());
+
+    let mut rabbit_client = test_client.clone();
+    rabbit_client.account_id = rabbit_account_id.clone();
+    rabbit_client.key_pair = rabbit_keys;
+
+    // Permission token for the trigger registration
+    // on behalf of alice
+    let permission_on_registration = PermissionToken::new(
+        "CanRegisterUserTrigger".parse().unwrap(),
+        &json!({ "account_id": "alice@wonderland", }),
+    );
+
+    // Trigger with 'alice' as authority
+    let trigger_id = TriggerId::from_str("alice_trigger")?;
+    let trigger = Trigger::new(
+        trigger_id.clone(),
+        Action::new(
+            Vec::<InstructionBox>::new(),
+            Repeats::Indefinitely,
+            alice_account_id.clone(),
+            ExecuteTriggerEventFilter::new()
+                .for_trigger(trigger_id.clone())
+                .under_authority(alice_account_id.clone()),
+        ),
+    );
+
+    // Register rabbit's account
+    test_client.submit_blocking(Register::account(rabbit_account))?;
+
+    let find_rabbit = FindAccountById {
+        id: rabbit_account_id.clone(),
+    };
+    test_client.request(find_rabbit).expect("Account not found");
+    info!("Rabbit is found.");
```

Comment:
This check is redundant imo, we check account registration ability in other test.

Reply:
This check was created only for debugging convenience and helped me a lot. So I would like to keep it if you don't mind.

Reply:
I don’t insist on removing this check, feel free to resolve discussion.